File: src/programa.c
#include <stdio.h>

int main() {
    char nombre[50];
    int anio, edad;

    // Solicitar el nombre
    printf("¿Cuál es tu nombre? ");
    scanf("%49[^\n]", nombre); // Lee hasta 49 caracteres o hasta un salto de línea

    // Solicitar el año de nacimiento
    printf("¿En qué año naciste? ");
    scanf("%d", &anio);

    // Calcular la edad
    edad = 2025 - anio;

    // Mostrar el mensaje
    printf("Hola %s, tienes %d años. Este programa está hecho en el lenguaje de programación: C.\n", nombre, edad);

    return 0;
}

File: src/programa.java
import java.util.Scanner;

public class Edad {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);

        System.out.print("¿Cuál es tu nombre? ");
        String nombre = sc.nextLine();

        System.out.print("¿En qué año naciste? ");
        int nacimiento = sc.nextInt();

        int edad = 2025 - nacimiento;

        System.out.println("Hola " + nombre + ", tienes " + edad + " años. Este programa está hecho en el lenguaje de programación: Java.");

        sc.close();
    }
}
